Horatio/Operations/AlertOperation.swift

```diff
@@ -38,8 +38,7 @@ class AlertOperation: Operation {
     // MARK: Initialization
 
     init(presentationContext: UIViewController? = nil) {
-        self.presentationContext = presentationContext ?? UIApplication.shared.keyWindow?.rootViewController
-
+        self.presentationContext = presentationContext
```

Comment:
What's the purpose of this? The original Apple code used this to ensure that a context would be used, even if no presentation context was currently set up.